src/redux/books/booksSlice.js

```diff
@@ -0,0 +1,50 @@
+import { createSlice } from '@reduxjs/toolkit';
+import { v4 as uuidv4 } from 'uuid';
+
+const initialState = {
+  bookItems: [
+    {
+      item_id: 'item1',
+      title: 'The Great Gatsby',
+      author: 'John Smith',
+      category: 'Fiction',
+    },
+    {
+      item_id: 'item2',
+      title: 'Anna Karenina',
+      author: 'Leo Tolstoy',
+      category: 'Fiction',
+    },
+    {
+      item_id: 'item3',
+      title: 'The Selfish Gene',
+      author: 'Richard Dawkins',
+      category: 'Nonfiction',
```

Comment:
- [ ] Based on instructions, you're to Include an array of books (initial state) which should be empty array not populated. Kindly follow this :

### Instructions:
![image](https://github.com/Unleashedicon/bookstore-1/assets/50990925/77a6dcb2-3880-4d0d-befb-11c482c43e73)